packages/kit/types.d.ts

```diff
@@ -1,3 +1,5 @@
+import { Logger } from './types.internal';
```

Comment:
It seems like `Logger` isn't really an internal type if it's being exposed here. Why not just put it in this file?

Reply:
it's internal insofar as you're not going to do 

```js
/** @type {import('@sveltejs/kit').Logger} */
```

— it's always just a property of the `builder` (which also needn't be exported) that's passed to the `adapt` function. this file should only export types that need to be consumed by users of Kit